internal/commands/buildpack_new.go

```diff
@@ -0,0 +1,73 @@
+package commands
+
+import (
+	"context"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack"
+	"github.com/buildpacks/pack/internal/dist"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/logging"
+)
+
+// BuildpackNewFlags define flags provided to the BuildpackCreate command
+type BuildpackNewFlags struct {
+	Path     string
+	Language string
+	Stacks   []string
+}
+
+// BuildpackCreator creates buildpacks
+type BuildpackCreator interface {
+	NewBuildpack(ctx context.Context, options pack.NewBuildpackOptions) error
+}
+
+// BuildpackNew generates the scaffolding of a buildpack
+func BuildpackNew(logger logging.Logger, client BuildpackCreator) *cobra.Command {
+	var flags BuildpackNewFlags
+	cmd := &cobra.Command{
+		Use:     "new <name>",
+		Short:   "Creates basic scaffolding of a buildpack.",
+		Args:    cobra.ExactValidArgs(1),
+		Example: "pack buildpack create my-buildpack",
+		Long:    "buildpack create generates the basic scaffolding of a buildpack repository.",
+		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+			path := flags.Path
+			if len(path) == 0 {
+				cwd, err := os.Getwd()
+				if err != nil {
+					return err
+				}
+				path = cwd
```

Comment:
The rfc seemed to indicate that we would create a directory `bar` in the current directory if none was provided (and the id was `foo/bar`). Is that the intention? This would just generate the files in the current directory, without creating a new sub-directory.